go/vt/orchestrator/inst/instance.go

```diff
@@ -48,10 +48,10 @@ type Instance struct {
 	LogSlaveUpdatesEnabled       bool // for API backwards compatibility. Equals `LogReplicationUpdatesEnabled`
 	LogReplicationUpdatesEnabled bool
 	SelfBinlogCoordinates        BinlogCoordinates
-	MasterKey                    InstanceKey
-	MasterUUID                   string
+	PrimaryKey                   InstanceKey
```

Comment:
It's a minor thing, but it's confusing: `PrimaryKey` can be confused with `PRIMARY KEY`. What do you think?

Reply:
Should I make it `PrimaryInstanceKey`, I think that would clear it up

Reply:
yeah, maybe? Or `SourceKey`? I'm really not sure.

Reply:
We can make it `SourceKey` too. That also makes sense. Then we should also rename to `SourceUUID` for consistency. WDYT?

Reply:
let's ask @deepthi

Reply:
Either is fine. In general using Source for replication related params makes sense.

Reply:
Alright i'll do that

Reply:
Done